support/build-debian.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+# Copyright 2024 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+DEB_ARCH="${1}"
+export GOOS="linux"
+case "${DEB_ARCH}" in
+    "armhf")
+        export GOARCH="arm"
+        export GOARM="6"
+        ;;
+    "arm64")
+        export GOARCH="arm64"
+        ;;
+    "amd64")
+        export GOARCH="amd64"
+        ;;
+    *)
+        echo "Usage: $0 <DEBARCH>" 1>&2
+        exit 1
+        ;;
+esac
+VERSION="$(git describe --tags --dirty)"
```

Comment:
Right! I just assumed the tag would always be present. Perhaps it would be better to let it fail if the repo is not fetched with tags? :thinking:

Reply:
Oh, so there are tags and I didn't fetch. I see.

Then I think it is a good idea what you suggest (to fail) but please add a check here in the shell script instead of letting it fail later in `dpkg-deb` command